### PR TITLE
Load ACS2010 from /data 

### PR DESCRIPTION
**Pull request by benhammondmusic**
- rather than needing to manually upload acs2010 files, this adds new functions that allow reading directly from the `/data` directory which are then pushed to BigQuery
- brings all functions in line to use `df` instead of some being `dataframe`
- use convention of `pd` for pandas

### CODE REVIEW

File: python/ingestion/gcs_to_bq_util.py [hunk 1]
```diff
@@ -221,15 +221,49 @@ def load_json_as_dataframe(gcs_bucket, filename, dtype=None):
...
+
+    #  go up a level
+    path_parent = os.path.dirname(os.getcwd())
+    os.chdir(path_parent)
+
+    # go into /data
+    os.chdir("data")
+
+    # combine sub-dir and filename
+    file_path = os.path.join(directory, filename)
```
Comment (joshzarrabi) [inline python/ingestion/gcs_to_bq_util.py:264]: Why change directories, why not just generate the entire filepath?

Comment (benhammondmusic) [reply python/ingestion/gcs_to_bq_util.py:264]: I couldn't get that way working.... and this did...  and I was tired of debugging it by running the entire process lol. If you have a quick fix I'm happy to use it

Comment (joshzarrabi) [reply python/ingestion/gcs_to_bq_util.py:263]: I'll give it a shot

**Review by joshzarrabi — APPROVED** [review]

File: python/ingestion/gcs_to_bq_util.py [hunk 2]
```diff
@@ -7,6 +7,9 @@
 from google.cloud import bigquery, storage
 
 
+DATA_DIR = os.path.join(os.sep, 'app', 'data')
```
Comment (benhammondmusic) [inline python/ingestion/gcs_to_bq_util.py:10]: will it always be `app` regardless of where this is running (docker locally, gcp, etc) ? That's what I wasn't sure

Comment (joshzarrabi) [reply python/ingestion/gcs_to_bq_util.py:10]: This code will only run in the docker environment

**Pull request merged**